gadget/validate.go

```diff
@@ -189,9 +189,17 @@ func validateReservedLabels(vs *VolumeStructure, reservedLabels []string) error
 	if vs.Label == "" {
 		return nil
 	}
-	if strutil.ListContains(reservedLabels, vs.Label) {
-		// a structure without a role uses one of reserved labels
-		return fmt.Errorf("label %q is reserved", vs.Label)
+	switch vs.Filesystem {
+	case "vfat":
```

Comment:
I'm not a fan of growing switches everywhere we need to deal with Label in gadget. I would prefer to add methods to the relevant structs

Reply:
Changed